docs/reference/module-options.md

```diff
@@ -41,13 +41,42 @@ require('apostrophe')({
 });
 ```
 
+## Using module options
+
+Most module options in core Apostrophe, and official extension modules, are used automatically for specific purposes. No additional work is needed to use them for their original purposes other than configuring them, if needed. In some cases it is helpful to reference the options object in custom module code. You can do this in any of the module configuration function sections using the `self` argument as `self.options`.
+
+For example, if you had a custom piece type, it might look like this:
+
+```javascript
+// modules/article/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  init (self) {
+    const moduleOptions = self.options;
+    // ...
+  },
+  methods (self) {
+    return {
+      logOptions () {
+        console.log('The module options are', self.options);
```

Comment:
This will include the entire apos object, which arrives as an option too, so it's a pretty intense thing to log. Maybe log a specific example.